### ISSUE TITLE
ModuleNotFoundError: No module named 'dateutil' with 0.4.8

### ISSUE THREAD

**Issue by mriedem** [fralau/mkdocs_macros_plugin#35]
Commit b8bcfafb59d23e734991ac03acd408a7d98ee272 adds a new runtime dependency on `dateutil` but that dependency is not listed in the `install_requires` list in setup.py:

https://github.com/fralau/mkdocs_macros_plugin/blob/b8bcfafb59d23e734991ac03acd408a7d98ee272/setup.py#L30

Which results in a failure to build docs with that version of the plugin:

```
Traceback (most recent call last):

  File "/home/travis/virtualenv/python3.6.7/bin/mkdocs", line 11, in <module>

    sys.exit(cli())

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/click/core.py", line 829, in __call__

    return self.main(*args, **kwargs)

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/click/core.py", line 782, in main

    rv = self.invoke(ctx)

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/click/core.py", line 1259, in invoke

    return _process_result(sub_ctx.command.invoke(sub_ctx))

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/click/core.py", line 1066, in invoke

    return ctx.invoke(self.callback, **ctx.params)

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/click/core.py", line 610, in invoke

    return callback(*args, **kwargs)

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/mkdocs/__main__.py", line 159, in build_command

    build.build(config.load_config(**kwargs), dirty=not clean)

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/mkdocs/config/base.py", line 197, in load_config

    errors, warnings = cfg.validate()

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/mkdocs/config/base.py", line 107, in validate

    run_failed, run_warnings = self._validate()

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/mkdocs/config/base.py", line 62, in _validate

    self[key] = config_option.validate(value)

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/mkdocs/config/config_options.py", line 130, in validate

    return self.run_validation(value)

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/mkdocs/config/config_options.py", line 591, in run_validation

    plgins[item] = self.load_plugin(item, cfg)

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/mkdocs/config/config_options.py", line 599, in load_plugin

    Plugin = self.installed_plugins[name].load()

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2346, in load

    return self.resolve()

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2352, in resolve

    module = __import__(self.module_name, fromlist=['__name__'], level=0)

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/macros/plugin.py", line 22, in <module>

    from .context import define_env

  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/macros/context.py", line 14, in <module>

    from dateutil.parser import parse as date_parse

ModuleNotFoundError: No module named 'dateutil'
```

**Comment by fralau** [comment]
Right. 

The quick fix is `pip install python-dateutil`  
(see [website](https://dateutil.readthedocs.io/en/stable/)).

**Comment by fralau** [comment]
Fixed in 593678b. Let me know?

**Comment by DominikaK** [comment]
Could you upload 0.4.9 to pypi?

**Comment by mriedem** [comment]
> 
> 
> Fixed in [593678b](https://github.com/fralau/mkdocs_macros_plugin/commit/593678b20f869058788ee4ba2185dd27ff38a32a). Let me know?

The fix looks fine. I'm just waiting for 0.4.9 to be on pypi so I can remove a blacklist in our travis automation.

**Comment by fralau** [comment]
It is now on pypi.

**Comment by mriedem** [comment]
> 
> 
> It is now on pypi.

Confirmed the issue is resolved with 0.4.9. Thanks!
